index.html

```diff
@@ -40,6 +40,112 @@ <h1>I’m Soban. Glad to see you!</h1>
+                    <img src="./images/Counter.svg" alt="counter">
+                    <p class="grey">2015</p>
+                </div>
+                <p class="workdesc">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="langs">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button class="see-project">See project</button>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio (3).svg" alt="Multi-Pozst" class="workimage">
+                <h2>Multi-Post Stories</h2>
+                <div class="desc">
+                    <p class="descA">CANOPY</p>
+                    <img src="./images/Counter.svg" alt="counter">
+                    <p class="grey">Back End Dev</p>
+                    <img src="./images/Counter.svg" alt="counter">
+                    <p class="grey">2015</p>
+                </div>
+                <p class="workdesc">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="langs">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button class="see-project">See project</button>
+            </div>
+        </div>
+        <div class="about">
+            <div class="outro">
+                <h2 class="outro-head">About Myself</h2>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me.</p>
+                <p id="connect">LET'S CONNECT</p>
+                <div class="social-links">
+                    <ul class="slinks olinks">
+                        <li><a href="https://twitter.com/SobanSyeed"><img src="./images/Vector.svg" alt="Twitter"></a>
+                        </li>
+                        <li><a href="https://www.linkedin.com/in/soban-syeed-82985b241/"><img
+                                    src="./images/Linkedin icon.svg" alt="LinkedIn"></a></li>
+                        <li><a href=""><img src="./images/Vector (2).svg" alt=""></a></li>
+                        <li><a href="https://github.com/SnakyMz"><img src="./images/Vector (1).svg" alt="Github"></a>
+                        </li>
+                        <li><a href=""><img src="./images/Vector (3).svg" alt=""></a></li>
+                    </ul>
+                </div>
+                <button class="see-project">Get my resume</button>
+                <ul class="skills">
+                    <li>Languages</li>
+                    <li>Frameworks</li>
+                    <li>Skills</li>
+                </ul>
```

Comment:
You have put in some quality amount of work in this section; however, you left out the arrow icons by the right of this items and also the languages items and I suggest including them to have a complete design as the template provided you.


Your design
<img width="218" alt="snip" src="https://github.com/SnakyMz/Portfolio/assets/84930355/e7140b82-4275-4e24-8e08-ddd614f6b063">

Figma

<img width="235" alt="hh" src="https://github.com/SnakyMz/Portfolio/assets/84930355/2b853cc6-c535-4e53-a7a2-722cb1fed769">

Reply:
Very well will do.